web-ext lint fails to convert path separators on Windows

### Is this a feature request or a bug?
Bug

### What is the current behavior?
(On _Windows_)
1. Clone https://github.com/birtles/web-ext-error
2. Run `npm test` (or just run `node_modules/.bin/web-ext lint`) from Git Shell or something similar

### What is the expected behavior?
```
$ npm test

> web-ext-error@1.0.0 test c:\Users\Brian\Documents\GitHub\web-ext-error
> web-ext lint

Validation Summary:

errors          0
notices         0
warnings        0
```

(This works if I move the icon to the same folder, or if I run the command on Linux.)

### What is the actual behavior?
```
$ npm test

> web-ext-error@1.0.0 test c:\Users\Brian\Documents\GitHub\web-ext-error
> web-ext lint

Validation Summary:

errors          3
notices         0
warnings        0

ERRORS:

Code                      Message                                     Description                                     File            Line   Column
MANIFEST_ICON_NOT_FOUND   An icon defined in the manifest could not   Icon could not be found at "images/icon.svg".   manifest.json
                          be found in the package.
MANIFEST_ICON_NOT_FOUND   An icon defined in the manifest could not   Icon could not be found at "images/icon.svg".   manifest.json
                          be found in the package.
MANIFEST_ICON_NOT_FOUND   An icon defined in the manifest could not   Icon could not be found at "images/icon.svg".   manifest.json
                          be found in the package.
npm ERR! Test failed.  See above for more details.
```

### Version information (for bug reports)

* **Firefox version**: 59
* **Your OS and version**: Windows 10
* Paste the output of these commands:

````
node --version && npm --version && node_modules/.bin/web-ext --version
````
v8.9.0
5.5.1
2.2.2

--- a/package.json
+++ b/package.json
@@ -49,7 +49,7 @@
   ],
   "dependencies": {
     "adbkit": "2.11.0",
-    "addons-linter": "0.31.0",
+    "addons-linter": "0.32.0",
     "babel-polyfill": "6.26.0",
     "babel-runtime": "6.26.0",
     "bunyan": "1.8.12",

